src/NormalDepthMap.cpp

```diff
@@ -1,139 +1,94 @@
-/*
- * NormalDepthMap.cpp
- *
- *  Created on: Mar 27, 2015
- *      Author: tiagotrocoli
- */
-
 #include "NormalDepthMap.hpp"
 
-#include <osg/Node>
+// C++ includes
+#include <iostream>
+#include <algorithm>
+
+// OSG includes
 #include <osg/Program>
 #include <osg/ref_ptr>
-#include <osg/Group>
 #include <osg/Shader>
 #include <osg/StateSet>
 #include <osg/Uniform>
 #include <osgDB/FileUtils>
+#include <osg/ShapeDrawable>
 
 namespace normal_depth_map {
 
-#define SHADER_PATH_FRAG "normal_depth_map/shaders/normalDepthMap.frag"
-#define SHADER_PATH_VERT "normal_depth_map/shaders/normalDepthMap.vert"
+#define SHADER_VERT "normal_depth_map/shaders/pass1.vert"
+#define SHADER_FRAG "normal_depth_map/shaders/pass1.frag"
 
-NormalDepthMap::NormalDepthMap(float maxRange, float maxHorizontalAngle, float maxVerticalAngle) {
-    _normalDepthMapNode = createTheNormalDepthMapShaderNode(maxRange, maxHorizontalAngle, maxVerticalAngle);
+NormalDepthMap::NormalDepthMap(float maxRange ) {
+    _normalDepthMapNode = createTheNormalDepthMapShaderNode(maxRange);
 }
 
-NormalDepthMap::NormalDepthMap(float maxRange, float maxHorizontalAngle, float maxVerticalAngle, float attenuationCoeff) {
-    _normalDepthMapNode = createTheNormalDepthMapShaderNode(maxRange, maxHorizontalAngle, maxVerticalAngle, attenuationCoeff);
+NormalDepthMap::NormalDepthMap(float maxRange, float attenuationCoeff) {
+    _normalDepthMapNode = createTheNormalDepthMapShaderNode(
+                                                          maxRange,
+                                                          attenuationCoeff);
 }
 
 NormalDepthMap::NormalDepthMap() {
     _normalDepthMapNode = createTheNormalDepthMapShaderNode();
 }
 
-void NormalDepthMap::setMaxRange(float maxRange) {
```

Comment:
Removeu o set and get do Range? Qual o motívo? Como fará pra atualizar em tempo de execução?